src/System.Collections.Concurrent/src/System.Collections.Concurrent.csproj

```diff
@@ -5,37 +5,30 @@
     <ProjectGuid>{EA610394-CBA3-4E5C-B3DB-AAEA7F640E89}</ProjectGuid>
     <AssemblyName>System.Collections.Concurrent</AssemblyName>
     <RootNamespace>System.Collections.Concurrent</RootNamespace>
+    <IsPartialFacadeAssembly>true</IsPartialFacadeAssembly>
   </PropertyGroup>
   <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'netcoreapp-Debug|AnyCPU'" />
   <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'netcoreapp-Release|AnyCPU'" />
   <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'uap-Windows_NT-Debug|AnyCPU'" />
   <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'uap-Windows_NT-Release|AnyCPU'" />
   <ItemGroup>
     <Compile Include="System\Collections\Concurrent\BlockingCollection.cs" />
-    <Compile Include="System\Collections\Concurrent\CDSCollectionETWBCLProvider.cs" />
-    <Compile Include="System\Collections\Concurrent\ConcurrentBag.cs" />
-    <Compile Include="System\Collections\Concurrent\ConcurrentDictionary.cs" />
-    <Compile Include="System\Collections\Concurrent\ConcurrentQueue.cs" />
-    <Compile Include="System\Collections\Concurrent\ConcurrentStack.cs" />
-    <Compile Include="System\Collections\Concurrent\IProducerConsumerCollection.cs" />
     <Compile Include="System\Collections\Concurrent\OrderablePartitioner.cs" />
     <Compile Include="System\Collections\Concurrent\Partitioner.cs" />
     <Compile Include="System\Collections\Concurrent\PartitionerStatic.cs" />
     <Compile Include="System\Collections\Concurrent\PlatformHelper.cs" />
-    <Compile Include="$(CommonPath)\System\Collections\Concurrent\ConcurrentQueue_Segment.cs">
-      <Link>Common\System\Collections\Concurrent\ConcurrentQueue_Segment.cs</Link>
-    </Compile>
   </ItemGroup>
-  <ItemGroup>
-    <Reference Include="System.Collections" />
-    <Reference Include="System.Diagnostics.Debug" />
-    <Reference Include="System.Diagnostics.Tools" />
-    <Reference Include="System.Diagnostics.Tracing" />
-    <Reference Include="System.Resources.ResourceManager" />
-    <Reference Include="System.Runtime" />
-    <Reference Include="System.Runtime.Extensions" />
-    <Reference Include="System.Threading" />
-    <Reference Include="System.Threading.Tasks" />
+  <ItemGroup Condition="'$(IsPartialFacadeAssembly)' == 'true'">
```

Comment:
We don't need to condition this ItemGroup since IsPartialFacadeAssembly is always set to true.